foolbox/models/base.py

```diff
@@ -64,11 +64,12 @@ def _preprocess(self, inputs: ep.TensorType) -> ep.TensorType:
         min_, max_ = self._model.bounds
         return x * (max_ - min_) + min_
 
-    def __getattr__(self, item: str) -> Any:
-        if item == "data_format":
-            return getattr(self._model, "data_format")
+    @property
+    def data_format(self) -> Any:
+        if hasattr(self._model, "data_format"):
+            return getattr(self._model, "data_format", None)
```

Comment:
This should be self._model.data_format (we already checked that it exists and Mypy shoud be abe to infer that)